pkg/notifiers/mailer.go

```diff
@@ -30,6 +32,17 @@ type Mailer struct {
 	SMTP
 }
 
+func getMessage(filters *models.Filters) string {
+	msg := "Vaccines are available in the following centers :"
+	if filters != nil {
```

Comment:
Filters will have default values always even if the user does not provide. This is true at least for the age. Could we remove this caluse.